examples/direct-upload/index.html

```diff
@@ -0,0 +1,24 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>Dwyl Direct Upload</title>
+    <meta http-equiv="content-type" content="text/html" charset="utf-8"/>
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/flat-ui/2.2.2/css/flat-ui.min.css">
+    <script type="text/javascript" charset="utf-8" src="./generate-keys.js"></script>
```

Comment:
WebPack has _many_ features we really only need _one of them_.
also: [![Coverage Status](https://coveralls.io/repos/github/webpack/webpack/badge.svg?branch=master)](https://coveralls.io/github/webpack/webpack?branch=master) https://coveralls.io/github/webpack/webpack

> Have you had a look at Roll Up? https://github.com/rollup/rollup#a-next-generation-es6-module-bundler